CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.11)
 
-project(marblerun VERSION 0.3.1)
+project(marblerun VERSION 0.3.2)
 execute_process(COMMAND bash "-c" "git rev-parse HEAD | tr -d '\n'" OUTPUT_VARIABLE GIT_COMMIT)
```

Comment:
The GIT_COMMIT injection is also kind of broken right now, I recently noticed.

It only gets/updates the GIT_COMMIT when it creates the build files (`cmake ..`), but not actually during build times. Meaning you end up with an outdated git commit string when you don't rebuild your build directory every single time.

Reply:
That's correct. However, this was best effort. If there is a better way to do it with cmake we should fix it?